x-pack/plugins/security_solution/public/detections/components/alerts_kpis/alerts_count_panel/index.test.tsx

```diff
@@ -249,8 +250,10 @@ describe('when isChartEmbeddablesEnabled = true', () => {
   beforeEach(() => {
     jest.clearAllMocks();
     mockUseQueryToggle.mockReturnValue({ toggleStatus: true, setToggleStatus: mockSetToggle });
-    mockUseIsExperimentalFeatureEnabled.mockReturnValueOnce(true); // for chartEmbeddablesEnabled flag
-    mockUseIsExperimentalFeatureEnabled.mockReturnValueOnce(false); // for alertsPageChartsEnabled flag
+    mockUseIsExperimentalFeatureEnabled.mockImplementation(
```

Comment:
Oh yea, you can just use this instead for the previous comment about mocking the experimental features fn too? I think it's a bit better as it's more explicit to your flag